### PR TITLE
some useful modifications and additions

### PR DESCRIPTION
**Pull request by hybridvamp**
user broadcast
broadcast with reply
broadcast formatting
total chat users count
ist in couple.py
slap module

### CODE REVIEW

**Comment by hybridvamp** [comment]
i have checked it with my bota and its working perfectly
i dont think its perfect
lemme know if there is any errors or mistakes

File: wbb/modules/slap.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# (c) @HYBRID_Bots
+
+import random
+import asyncio
+from wbb import app # This is bot's client
+from wbb import app2 # userbot client, import it if module is related to userbot
```
Comment (TheHamkerCat) [inline wbb/modules/slap.py]: why did you import userbot client?

Comment (TheHamkerCat) [reply wbb/modules/slap.py]: Also, this "slap" module is unneeded, so you can remove it

**Review by TheHamkerCat — CHANGES_REQUESTED** [review]

**Comment by hybridvamp** [comment]
removed slap

**Review by TheHamkerCat — APPROVED** [review]

**Pull request merged**